prow/tide/tide.go

```diff
@@ -1516,6 +1499,8 @@ func prHasSuccessfullTideStatusContext(pr CodeReviewCommon) bool {
 
 // tryMerge attempts 1 merge and returns a bool indicating if we should try
 // to merge the remaining PRs and possibly an error.
+//
+// tryMerge is used by mergePRs only, referenced by GitHubProvider only.
 func tryMerge(mergeFunc func() error) (bool, error) {
```

Comment:
Please collocate these with the GitHubProvider so that this file is only common logic.

Reply:
will do once this PR is approved